draft-ietf-rats-msg-wrap.md

```diff
@@ -637,6 +638,13 @@ Client-authenticated TLS may be used to bind a Collection CMW of Evidence messag
 However, the client key used with TLS should not be that of the end-user or owner of the device.
 Instead, it should be attestation-oriented key material from the device or the attester manufacturer.
 
+## CMW in X.509 {#seccons-x509}
+
+Publishing attestation information in a public certificate raises privacy and, in some cases, security concerns.
+For instance, an individual seeking a publicly-trusted code signing certificate may be willing to disclose the details of the hardware where their code signing keys are stored (e.g., HSM model, patch level, etc.).
+However, they likely do not want this information to be publicly accessible.
+Applications that intend to include attestation data in X.509 certificates should define a Certificate Practices Statement that clearly defines the circumstances under which the Certificate Authority (CA) can include attestation data in certificates.
```

Comment:
Note that the privacy implications of publishing attestation results in a Web Token or any other structure besides a certificate are similar/same as with certificates. The wording shouldn't bias certificates. Possibly, the better approach is to create a section on privacy considerations and address the topic more generally?

Reply:
Mike’s concern is about evidence claims that should remain private between the attester and the verifier. There’s a risk of unwarranted disclosure if the verifier includes them in an x509 cert. It’s not about DICE-style certs.

Reply:
Right, exactly. Once you start talking about CSRs and X.509, there's an assumption that the contents of the CSR just get dumped over into the X.509 cert. I just think it needs to be called out that certain types of attestation data (mainly evidence, but probably also endorsements and ARs) are 100% reasonable to put in a CSR, and 100% unreasonable to put in an X.509 cert.

In the LAMPS CSR-Attest draft, we have danced around this issue very carefully and very clearly stated that this is a CSR mech and not an X.509 mech ... we recognize the need for that, but we will address that in another document that has space to treat the security and privacy implications properly.

Reply:
> […] Possibly, the better approach is to create a section on privacy considerations and address the topic more generally?

[b6f21d4](https://github.com/ietf-rats-wg/draft-ietf-rats-msg-wrap/pull/189/commits/b6f21d4ace5c27b3c7cfc0191ee0f594fded0509) takes a stab at it.